### DISCUSSION TITLE
A list of 2024 meetings from the GBIF North America Steering Committee

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#7238] [Announcements]
This is not exhaustive by any means, but I thought Arctos people might be interested. These are some meetings that various members of the GBIF NA Steering Committee may attend (if funding is available). I am specifically tagging people who I think might be interested.

January 23, 2024:  [the Summit on Ocean Biodiversity](https://science.si.edu/in-the-ocean/summit-on-ocean-biodiversity/) @happiah-madson @falco-rk 
January 24: [Utilizing Cultural Intelligence to advance Tribal engagement and partnerships](https://cires.colorado.edu/events/utilizing-cultural-intelligence-advance-tribal-engagement-and-partnerships) @ArctosDB/diversity-and-inclusion 
January:  [ESIP: Grounded in Trust: Data Ethics Empower Collaboration](https://www.esipfed.org/meetings)
February 18-23: [Ocean Sciences](https://www.agu.org/ocean-sciences-meeting) in New Orleans, LA @happiah-madson @falco-rk 
May 29-31: [Digital Data](https://www.idigbio.org/content/digital-data-biodiversity-research-conference-2024) at KU
June: [National eDNA workshop](https://secwww.jhuapl.edu/EventLink/Event/300) in Maryland @happiah-madson @falco-rk @campmlc @megulbranson 
July: [ESIP](https://www.esipfed.org/) no date yet
August 4-9: [ESA](https://www.esa.org/longbeach2024/) in August (Humboldt Core) in Long Beach, CA
September 2-6: [SPNHC/TDWG](https://spnhc.org/events/event/spnhc-tdwg-2024/) in Okinawa

If I learn about more as the year goes on, I will add them here.

**Comment by ewommack** [comment]
I might be able to make the 24th January one remotely. I've added it to my calendar, and we'll see how the volunteer schedule takes off at the UWYMV.
